Automattic/wp-calypso
# Clear 'unknown attribute' errors in Calypso packages where it is safe to do so

Fixes #60677

Related to #64984

#### Proposed Changes

Run the 'unknown attribute' check in WebStorm and apply fixes, where it looks safe to do so. 

The changes mainly fall into two categories: using DOM name instead of JSX name, e.g. `class` instead of `className` or using the wrong case e.g. `fill-rule` instead of `fillRule` as in #60677 which lead to this PR. 

In addition, the `href` attribute was also removed from **client/components/post-schedule/docs/example.jsx**, where it is invalid and isn't referenced in the file or by the component (copy/paste error assumed).

#### Testing Instructions

As well as the automated tests, it should be enough to review the changes one by one. Functionally nothing has changed except the errors mentioned above are now gone. Before this change you would see a message in the log like this:

![image](https://user-images.githubusercontent.com/6851384/178272546-f5e57350-25b1-4355-85bf-7233741adcf6.png)
